projects/plugins/boost/app/modules/optimizations/image-cdn/src/liar.ts

```diff
@@ -0,0 +1,6 @@
+import { dynamicSrcset } from './srcset';
+
+( function () {
+	const lazyImages = document.querySelectorAll( 'img[loading=lazy]' );
```

Comment:
Do we want to filter out the images where `image.complete === true`. For images that are already loaded, perhaps we can check `currentSrc` and make sure the image isn't smaller than the target size?

Reply:
I think I tried that at some point, but the results weren't satisfactory - `image.complete` is true only when the image has loaded - there's no way to actually detect whether the image is actually loading - so there's very little benefit of doing that at the moment.

As for currentSrc - I wanted to do that as well, but since this grew a little out of what I initially expected, I think it can be a follow-up improvement.